addons/mail/static/src/js/views/activity/activity_renderer.js

```diff
@@ -208,7 +208,8 @@ var ActivityRenderer = AbstractRenderer.extend({
                 var def = widget.appendTo($td).then(function () {
                     // replace clock by closest deadline
                     var $date = $('<div class="o_closest_deadline">');
-                    var date = new Date(activity_group.o_closest_deadline);
+                    var split_date = activity_group.o_closest_deadline.split('-');
+                    var date = new Date(split_date[0], split_date[1]-1, split_date[2]);
```

Comment:
By convention, we camelcase variable names in JS.

I'd add a comment to explain:
 1) why you split the year/month/day to create a date (to have a date, no tz, instead of a datetime, tz dependant)
 2) why you do -1 for the month

Native dates can't be mocked, so you can't test it as it is. Typically, we use moment (always UTC), and manually apply the TZ offset from the session, so that we can mock it in tests (for instance, to simulate that we are in mexico TZ). But it would require a larger diff as this is not how the activity renderer was designed.